buildtools/gocmd/go.go

```diff
@@ -71,17 +69,14 @@ func (g *Go) ListOne(pkg string) (Package, error) {
 }
 
 // List runs `go list` to return information about packages.
-func (g *Go) List(pkgs []string) ([]Package, error) {
+func (g *Go) List(flags, pkgs []string) ([]Package, error) {
```

Comment:
[nit] Same here: I personally prefer `(pkgs, flags []string)`, because it feels like `pkgs` is the "primary" argument somehow.